plugins/wpe-content-model/tests/integration/content-registration/example-data/expected-post-types.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * Example data for registering custom post types and custom fields.
+ *
+ */
+return array (
+	'dog' =>
+		array (
+			'name' => 'dog',
+			'label' => 'Dogs',
+			'singular_label' => 'Dog',
+			'description' => '',
+			'public' => true,
+			'publicly_queryable' => true,
+			'show_ui' => true,
+			'show_in_nav_menus' => true,
+			'delete_with_user' => false,
+			'show_in_rest' => true,
+			'has_archive' => true,
+			'has_archive_string' => '',
+			'exclude_from_search' => false,
+			'capability_type' => 'post',
+			'hierarchical' => false,
+			'rewrite' => true,
+			'rewrite_slug' => '',
+			'rewrite_withfront' => true,
+			'query_var' => true,
+			'query_var_slug' => '',
+			'show_in_menu' => true,
+			'supports' =>
+				array (
+					'title',
+					'editor',
+					'thumbnail',
+					'custom-fields',
+				),
+			'taxonomies' =>
+				array (
+				),
+			'labels' =>
+				array (
+					'menu_name' => 'Dogs',
+					'all_items' => 'All Dogs',
+					'add_new' => 'Add new',
+					'add_new_item' => 'Add new Dog',
+					'edit_item' => 'Edit Dog',
+					'new_item' => 'New Dog',
+					'view_item' => 'View Dog',
+					'view_items' => 'View Dogs',
+					'search_items' => 'Search Dogs',
+					'not_found' => 'No Dogs found',
+					'not_found_in_trash' => 'No Dogs found in trash',
+					'parent' => 'Parent Dog:',
+					'featured_image' => 'Featured image for this Dog',
+					'set_featured_image' => 'Set featured image for this Dog',
+					'remove_featured_image' => 'Remove featured image for this Dog',
+					'use_featured_image' => 'Use as featured image for this Dog',
+					'archives' => 'Dog archives',
+					'insert_into_item' => 'Insert into Dog',
+					'uploaded_to_this_item' => 'Upload to this Dog',
+					'filter_items_list' => 'Filter Dogs list',
+					'items_list_navigation' => 'Dogs list navigation',
+					'items_list' => 'Dogs list',
+					'attributes' => 'Dogs attributes',
+					'name_admin_bar' => 'Dog',
+					'item_published' => 'Dog published',
+					'item_published_privately' => 'Dog published privately.',
+					'item_reverted_to_draft' => 'Dog reverted to draft.',
+					'item_scheduled' => 'Dog scheduled',
+					'item_updated' => 'Dog updated.',
+					'parent_item_colon' => 'Parent Dog:',
+				),
+			'show_in_graphql' => true,
+			'graphql_single_name' => 'dog',
+			'graphql_plural_name' => 'dogs',
+			'fields' => array(
+				'dog-test-field' => array(
+					'type' => 'string',
+					'description' => 'dog-test-field description',
+					'show_in_rest' => true,
+				),
+				'another-dog-test-field' => array(
+					'type' => 'string',
+					'description' => 'another-dog-test-field description',
+					'show_in_rest' => false,
+				),
+			),
```

Comment:
Thanks for adding these — really helps to understand intended use.

Also reminds me we need to look into generating all those labels automatically, since we're probably only going to ask users for one version of the label.